### PR TITLE
Update Latest Attestations Use Committee Cache

### PR DESCRIPTION
**Pull request by terenc3t**
Every time we update latest attestations we call `helpers.CrosslinkCommitteesAtSlot`, this is a very tedious/repetitive process we could easily eliminate by using the committee cache. This PR makes use of the committee cache whenever we update the latest attestation

### CODE REVIEW

File: beacon-chain/core/helpers/committee.go [hunk 1]
```diff
@@ -660,9 +658,9 @@ func crosslinkCommittees(state *pb.BeaconState, input *shufflingInput) ([]*Cross
 	return crosslinkCommittees, nil
 }
 
-// toCommitteeCache coverts crosslink committee object
+// ToCommitteeCache coverts crosslink committee object
```
Comment (prestonvanloon) [inline beacon-chain/core/helpers/committee.go]:
```suggestion
// ToCommitteeCache converts crosslink committee object
```

File: beacon-chain/attestation/service.go [hunk 2]
```diff
@@ -163,7 +166,9 @@ func (a *Service) handleAttestation(ctx context.Context, msg proto.Message) erro
 // have been noted in the attestation pool. If not, it updates the
 // attestation pool with attester's public key to attestation.
 func (a *Service) UpdateLatestAttestation(ctx context.Context, attestation *pb.Attestation) error {
+	var err error
```
Comment (prestonvanloon) [inline beacon-chain/attestation/service.go]: Why is this needed?

Comment (prestonvanloon) [reply beacon-chain/attestation/service.go]: Still wondering why :)

Comment (terenc3t) [reply beacon-chain/attestation/service.go]: it's not needed, I forgot to push the latest commit : )
